app/views/gardens/_thumbnail.html.haml

```diff
@@ -1,36 +1,21 @@
 .panel.panel-success
   .panel-heading
     %h3.panel-title
-      = link_to "#{garden.owner.login_name}'s garden", garden
+      = link_to garden.name, garden
```

Comment:
I prefer this; with maybe a fallback for garden name = blank (or we add model validations to require it if not already there)